packages/tools/examples/colorbar/index.ts

```diff
@@ -101,6 +102,21 @@ addDropdownToToolbar({
   },
 });
 
+// Dropdown that allows the user to change colorbar position
+addDropdownToToolbar({
+  labelText: 'Colorbar Position: ',
+  options: {
+    values: ['right', 'bottom'],
+    defaultValue: 'right',
+  },
+  style: {
+    maxWidth: '120px',
+  },
+  onSelectedValueChange: (position) => {
+    updateColorbarPosition(position as string);
```

Comment:
This should go through a command function so that it can be tied to a keyboard or other action such as onLoad in a hanging protocol.

Reply:
This is just an example in cornerstone3D, not OHIF. Its purpose is to demonstrate the minimum code needed to test different functionalities in `cs3d`. I don't want to create a HP engine for cs3d